modules/Data.py

```diff
@@ -114,6 +114,11 @@ def update_conversion_rates(output_currency, json_output_enabled):
                 log.log_error("Failed to find the exchange rate for outputCurrency {0}!".format(output_currency))
                 log.log_error("Make sure that {0} is either traded on Poloniex or supported by blockchain.info: {1}"\
                               .format(output_currency, "https://blockchain.info/api/exchange_rates_api"))
+                log.log_error("Using BTC as the output currency".format(url))
+                update_conversion_rates('BTC', json_output_enabled)
+            except:
```

Comment:
1. Because otherwise we get a log message saying to open a github issue to fix something we can't fix (Connection error usually). The first one catches ValueError which occurs when you put a wrong currency value in. The second catches everything else. 

2. As above, I can change it if you like.